### ISSUE TITLE
Concatenate Tag Cloud on Libraries page.

### ISSUE THREAD

**Issue by Jafner** [Floppy/van_dam#886]
From the scan pulling pieces of the filename and converting them into tags, I have 876 tags right now, and they are all listed on the Libraries page. This looks pretty ridiculous.  
![image](https://user-images.githubusercontent.com/40403594/208215714-4388c7c3-303b-4b55-b189-b9a00e805412.png)  

It would be nice if this could be concatenated to match the bottom of the models list, and maybe sorted by something (models tagged, random, creation date, or alphabetical, etc.).

**Comment by Floppy** [comment]
👍

**Comment by ksuquix** [comment]
A few notes on this one.   
#1063 added sorting via alphabetical or tag count (with ability to display tag count).  Also adjustable trim on count threshold
#963 added a bunch of sorting, etc that might make need of some tags less (can trim tags because sort works)
#948 can turn off auto-tagging
#1054 makes separate libraries have separate tag clouds if different libraries get selected

I found the auto-tagger to be pretty bad for my uses, and I'm using #967 to auto-make collection / creator, and turned off auto tagging.  (then declared tag bankruptcy, deleted all my tags, and started tagging with my own more ordered system).

**Comment by Floppy** [comment]
@ksuquix do you reckon your tag improvements in v0.42.0 add up to addressing this?

**Comment by ksuquix** [comment]
It seems like it should be addressed now, unless we get more direction after all the changes.  I expect @Jafner will need to review how everything is working now and open a new issue if needs aren't covered now.

**Comment by Jafner** [comment]
Finally had a chance to check this out. I love all of the improvements to navigability.  

I suppose at this point it's an aesthetic issue, rather than a pragmatic one. The tag cloud extends 2-3x farther down the page than the models list (at 9 items on my 1440p display). The tags do appear to be sorted by number of models tagged now, which is awesome for making the tag cloud itself more useful.   

I think simply collapsing the cloud with a "View All" button would clean things up nicely. But I also understand it's not really a big issue, just visual polish.  

I've attached an image of what the bottom of my models list looks like:  

![image](https://user-images.githubusercontent.com/40403594/224775589-840dcd5f-8a55-4bf3-88b7-ff91c01aa8d7.png)

**Comment by Floppy** [comment]
Yeah, I've been looking at my own which is now larger, because 1-element tags are shown, and I feel what you're after here a bit more. I will take a look at making the list scrollable, perhaps.

**Comment by ksuquix** [comment]
There's a setting for the number of minimum taggings to show up in the list
now.   It defaults to zero, but you guys who have tons of tags might want
to set it higher.

**Comment by Floppy** [comment]
I'm not sure I can solve this without switching over to CSS Grid (see #2612)
